### PR TITLE
FEAT: Connection with Firebase

### PR DESCRIPTION
**Pull request by aciobotaruSV**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+              <Link to="/">Login</Link>
+            </li>
+            <li>
+              <Link to="/dashboard">Dashboard</Link>
+            </li>
+          </ul>
+        </nav>
+        <Switch>
+          <Route path="/auth">
+            <AuthForm />
```
Comment (rolandkasa-95) [inline src/App.js:23]: Make it a little cleaner, put the component call inside the route.

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+          </Route>
+          <Route path="/dashboard">
+            <Dashboard />
```
Comment (rolandkasa-95) [inline src/App.js:26]: Same here.

File: src/FireStoreData.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { useState, useEffect } from "react";
+import { collection, getDocs } from "firebase/firestore/lite";
+
+const FireStoreData = (database) => {
```
Comment (rolandkasa-95) [inline src/FireStoreData.js:4]: Does this needs to be a fully fledged component?

File: src/firebase.config.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import { initializeApp } from "firebase/app";
+import { getFirestore } from "firebase/firestore/lite";
+
+const firebaseConfig = {
```
Comment (rolandkasa-95) [inline src/firebase.config.js:4]: Move the initialization to a service file.

**Review by rolandkasa-95 — CHANGES_REQUESTED** [review]

**Comment by aciobotaruSV** [comment]
Please review

File: src/components/AuthForm/AuthForm.js [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+        if (credential) {
+          // This gives you a GitHub Access Token. You can use it to access the GitHub API.
+          const token = credential.accessToken;
+          // ...
+          console.log(token);
+        }
+
+        // The signed-in user info.
+        const user = result.user;
+        console.log(user.email);
```
Comment (rolandkasa-95) [inline src/components/AuthForm/AuthForm.js:43]: Remove console logs.

File: src/components/AuthForm/AuthForm.js [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+        // The AuthCredential type that was used.
+        const credential = GithubAuthProvider.credentialFromError(error);
+        // ...
+      });
+    return () => {
+      setLoggedIn(false);
+    };
+  }, []);
+
+  console.log(loggedIn);
```
Comment (rolandkasa-95) [inline src/components/AuthForm/AuthForm.js:62]: Same here.

**Review by rolandkasa-95 — APPROVED** [review]

**Pull request merged**